app/uk/gov/hmrc/gform/controllers/AuthenticatedRequestActions.scala

```diff
@@ -281,47 +316,48 @@ class AuthenticatedRequestActions(
 
     authorised(predicate)
       .retrieve(defaultRetrievals) {
-        case authProviderId ~ enrolments ~ affinityGroup ~ internalId ~ externalId ~ userDetailsUri ~ credentialStrength ~ agentCode =>
+        case authProviderId ~ enrolments ~ _ ~ internalId ~ externalId ~ userDetailsUri ~ credentialStrength ~ agentCode =>
```

Comment:
instead of ignoring affinity group like this `~ _ ~` it would be better not to ask for it in the first place by modifying `defaultRetrievals`

Reply:
Sorted